Assets/Scripts/Interactables/LaunchArcRenderer.cs

```diff
@@ -87,4 +61,19 @@ Vector3 CalculateArcPoint(float t, float maxDistance)
         float y = x * Mathf.Tan(radianAngle) - ((g * x * x) / (2 * velocity * velocity * Mathf.Cos(radianAngle) * Mathf.Cos(radianAngle)));
         return new Vector3(x, y);
     }
+
+    private void OnEnable()
+    {
+        // Allows arc to be rendered in correct position when re-enabled
+        Update();
+    }
+
+    private void Update()
+    {
+        mousePosition = GameManager.Instance.GetThrowController().GetMousePosition();
+        transform.LookAt(mousePosition);
+        transform.Rotate(0, 270, 0);
```

Comment:
The logic being here is kind of weird for me. Feels like something the ThrowController should manage since it's managing everything else related to this... thoughts?